src/DurableTask.SqlServer/Scripts/logic.sql

```diff
@@ -478,6 +478,20 @@ BEGIN
 END
 GO
 
+CREATE OR ALTER PROCEDURE dt.RewindInstance
```

Comment:
If it's not strictly necessary, I'd prefer we remove `dt.RewindInstance` and remove `dt._RewindInstance`. This is mainly because I want to avoid having people take a dependency on this sproc directly. In the future, we may change the implementation to be backend-agnostic, in which case we'd need to deprecate this "public" stored procedure.

Reply:
Oops, I had a typo in my comment. I meant to suggest we keep just the private sproc. I’m fixing the comment to reflect this.

Reply:
Switched to two private proc (have to keep both as one is responsible for the transaction, which we do not want to reopen recursively.

Reply:
Got it - thanks for clarifying this point. I didn't realize this is why you had two sprocs.